stocks/tests.py

```diff
@@ -0,0 +1,52 @@
+"""This module is for testing stocks"""
+from django.test import TestCase
+from .models import Stock, DailyStockQuote
+
+
+class StocksViewTests(TestCase):
+    """
+    Testing Stocks Model
+    """
+
+    def test_api_for_real_stock(self):
+        """
+        Testing adding stock via endpoint, asserting stock is inserted
+        """
+        ticker = "googl"
+        name = "Google"
+        data = {'name': name, 'ticker': ticker}
+        request = self.client.post('/stocks/addstock/', data)
```

Comment:
Could you mock the API that actually makes a request to Yahoo? That makes us more resilient to failures and allows better assertions. Alternatively stop the trigger as shown here (https://github.com/Neitsch/ASE4156/blob/master/tests/test_viewer_graphql.py#L20)